ambry-store/src/main/java/com/github/ambry/store/CompactionLog.java

```diff
@@ -380,6 +385,35 @@ byte to indicate whether startOffsetOfLastIndexSegmentForDeleteCheck is present
     }
   }
 
+  @Override
+  public String toString() {
+    StringBuffer buffer = new StringBuffer();
```

Comment:
I think you wanted `StringBuilder` here.

Reply:
That's right, string builder should be much faster.